cram2bam

```diff
@@ -10,13 +10,10 @@ fi
 cram=$1
 
 ## use 80 percent of available threads
-threads=$( expr $(grep -c processor /proc/cpuinfo) / 10 \* 8)
-samtools="/data/tools/samtools_v1.10/samtools"
+#threads=$( expr $(grep -c processor /proc/cpuinfo) / 10 \* 8)
 
-## reset paths for use at datastore server
-if [[ "$(hostname)" == "datastore" ]]; then
-    samtools="/data/common/tools/samtools_v1.10/samtools"
-fi
+threads=3
```

Comment:
In general its best to make the image as independent from the machine as possible. So we should either find a way to make the cpu detection magic work or allow the caller pass the threads (fine to default to 3)

Reply:
Agreed. Noticed that pigz defaults to using all available so might not need specific thread count anyway.

Reply:
Also to add some colour to that, in the pipeline we have a few spots where we need to specify a count and we just use a little bit of shell code to determine the number of cores. Then we control what that really means by manipulating the sizes of the machines in configuration.